Bio/PDB/Atom.py

```diff
@@ -76,19 +76,24 @@ def __init__(self, name, coord, bfactor, occupancy, altloc, fullname, serial_num
     # Sorting Methods
     # standard across different objects and allows direct comparison
     def __eq__(self, other):
+        """Test equality."""
         if isinstance(other, Atom):
-            return (self.id, self.altloc) == (other.id, other.altloc)
+            return self.full_id == other.full_id
         else:
             return NotImplemented
 
     def __ne__(self, other):
+        """Test inequality."""
         if isinstance(other, Atom):
-            return (self.id, self.altloc) != (other.id, other.altloc)
+            return  self.full_id != other.full_id
```

Comment:
Two spaces after ``return``, should be one - that's giving this error on the style checks:

```
Bio/PDB/Atom.py:88:19: E271 multiple spaces after keyword
```